src/include/daos_api.h

```diff
@@ -532,6 +532,17 @@ int
 daos_cont_alloc_oids(daos_handle_t coh, daos_size_t num_oids, uint64_t *oid,
 		     daos_event_t *ev);
 
+/**
+ * Trigger aggregation to specified epoch
+ *
+ * \param[in]	coh	Container handle
+ * \param[in]	epoch	Epoch to be aggregated to.
+ * \param[in]	ev	Completion event, it is optional and can be NULL.
+ *			The function will run in blocking mode if \a ev is NULL.
+ */
+int
+daos_cont_aggregate(daos_handle_t coh, daos_epoch_t epoch, daos_event_t *ev);
```

Comment:
That's not to me flexible, and in most case, aggregating to current time isn't safe and efficient if there are active ongoing IOS with epoch very close to the aggregation epoch.

If user really want to aggregate to current time, he can just use a script to convert current time to an epoch.

Reply:
I'll update the tool 'dcont' to aggregate to current time when epoch isn't specified.

Reply:
OK maybe there could be use cases where user would aggregate something like an epoch before the last hour, or at a certain point in time in the past.
In any way, i think it would still help if you provide a way to trigger at the current epoch of server by just passing epoch = 0.

Reply:
Yes, I've updated the tool 'dcont' to aggregate to current time by default when epoch isn't specified.

It looks to me that current time should be passed by caller, it could be a simple 'clockgettime' or a complex 'get current HLC after HLC sync', or whatever, we don't have to integrate it into this simple aggregation trigger API.

Reply:
the issue is that the client and server clocks can be out of sync, and since most updates and fetch use server clock today, we might need to have this time set on server side.
I already have an API to convert time to epoch - daos_ts2epoch() - but on client side this can be out of sync than on server side.

Reply:
Yes, I overlooked that this is a client side API. I'll update it and take epoch 0 as 'current time'.